ipld_missing.go

```diff
@@ -0,0 +1,19 @@
+package ipldgit
+
+import (
+	"github.com/ipfs/go-cid"
+	cidlink "github.com/ipld/go-ipld-prime/linking/cid"
+)
+
+func (ip _Link__Prototype) fromString(l *_Link, s string) error {
+	c, err := cid.Decode(s)
+	if err != nil {
+		return err
+	}
+	l.x = cidlink.Link{Cid: c}
+	return nil
+}
+
+func (ip *_Link) String() string {
+	return ip.String()
```

Comment:
Hm, I don't follow how this is useful or why we need it.

Reply:
i think it was for debugging, but doesn't seem harmful.